pkg/controllermanager/apis/config/v1alpha1/types.go

```diff
@@ -96,6 +98,17 @@ type ControllerManagerControllerConfiguration struct {
 	ManagedSeedSet *ManagedSeedSetControllerConfiguration `json:"managedSeedSet,omitempty"`
 }
 
+// BastionControllerConfiguration defines the configuration of the Bastion
+// controller.
+type BastionControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on
+	// events.
+	ConcurrentSyncs int `json:"concurrentSyncs"`
+	// MaxLifetime is the maximum time a Bastion resource can exist before it is
+	// forcefully deleted.
+	MaxLifetime time.Duration `json:"maxLifetime"`
```

Comment:
Should this be optional and defaulted?

Reply:
I made it optional now; I assume it's already defaulted by the `defaults.go`, or is there more defaulting action somewhere else?

Reply:
Nope, that's it - looks good, thanks!